src/infrastructure/getNodePeers.js

```diff
@@ -22,14 +22,12 @@ import dto from './dto'
 import format from '../format'
 
 class sdkNode {
-    static getNodes = async () => {
+    static getNodePeers = async () => {
       const path = `/node/peers`
       const response = await axios.get(http.nodeUrl + path)
 
       const nodes = response.data.map(node => dto.createNodeInfoFromDTO(node, http.networkType))
 
-      console.log(format.formatNodesInfo(nodes))
-
       return format.formatNodesInfo(nodes)
     }
 }
```

Comment:
i dont think this has to be a class, can be just exporting the function: 

```
export getNodePeers = async () => {
    // ...
}
```

and then all imports just become :

```
import {getNodePeers} from '../infrastructure/getNodePeers'
```

Reply:
Since this class is totally static there is no sense in it. I agree with Greg it can be simple export of functions. But we have whole infrastructure in this static-class approach. I think it is make sense to keep everything in one style whether single functions or static-class.

What do you think @evias  @AnthonyLaw ?

Reply:
I think yes, we should keep everything in one style and using function approach.